Content/posts/swift-package-index-metadata-file.md

```diff
@@ -0,0 +1,103 @@
+---
+date: 2020-10-30 12:00
+title: The Swift Package Index Metadata File – First Steps
+description: …
+---
+
+The vast majority of the information you see in [the package index](https://swiftpackageindex.com) is generated from a single URL, the location of the package’s git repository.
+
+From that URL, we’re able to gather a huge amount of metadata. We gather data from the repository itself, the package manifest, the GitHub API (yes, it’s GitHub only for now, but [we do plan to support other hosts](https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/issues/485)), and from running builds of the package to check compatibility with operating systems and Swift versions.
```

Comment:
I feel like `huge amount` sounds a bit big, perhaps `a lot of`? Just a nit though.

Reply:
Yep, the whole sentence was a bit unnecessary actually. Changed to:

> From that URL we gather data from the repository itself, ...